scapy/contrib/bgp_fields.py

```diff
@@ -0,0 +1,347 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""BGP-4 disector: fields for implementing MPBGP extensions"""
+
+import socket
+if not socket.has_ipv6:
+    raise socket.error("can't use AF_INET6, IPv6 is disabled")
```

Comment:
Will your module refuse to work without IPv6?

BTW, Scapy has a configuration attribute to check whether IPv6 is enabled: `conf.ipv6_enabled`